### PR TITLE
Prepare Core and Identity releases

### PR DESCRIPTION
**Pull request by antkmsft**
After #2526, #2525, #2524, #2523, and #2448 go in, and as we are ready to release, this PR goes in.
Edit: ideally, also #2532.

### CODE REVIEW

File: sdk/identity/azure-identity/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 1.1.0-beta.1 (Unreleased)
+## 1.1.0-beta.1 (2021-07-01)
```
Comment (RickWinter) [inline sdk/identity/azure-identity/CHANGELOG.md:3]: Add an entry for the new feature, MSI Credential

File: sdk/core/azure-core/CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,10 @@
 # Release History
 
-## 1.1.0 (Unreleased)
+## 1.1.0 (2021-07-01)
+
+### New Features
+
+- Added internal support for Azure Key Vault client library.
```
Comment (RickWinter) [inline sdk/core/azure-core/CHANGELOG.md:7]: Put this in other changes.  Its not customer facing or affecting so calling it out in the release notes does NOT make sense

**Review by Jinming-Hu — APPROVED** [review]

File: sdk/identity/azure-identity/CHANGELOG.md (hunk 1 above)
Comment (antkmsft) [reply sdk/identity/azure-identity/CHANGELOG.md:3]: ManagedIdentityCredential PR adds it.

Comment (antkmsft) [reply sdk/identity/azure-identity/CHANGELOG.md:3]: See just below now.

File: sdk/core/azure-core/CHANGELOG.md [hunk 3]
```diff
@@ -1,11 +1,15 @@
 # Release History
 
-## 1.1.0 (Unreleased)
+## 1.1.0 (2021-07-01)
 
 ### Bug Fixes
 
 - Fixed a memory leak issue in `Base64Encode()`. (A community contribution, courtesy of _[jorgen](https://github.com/jorgen)_)
 
+### Other Changes and Improvements
```
Comment (ahsonkhan) [inline sdk/core/azure-core/CHANGELOG.md:9]:
```suggestion
### Other Changes
```


The headings we want to use are:
`Features Added`, `Breaking Changes`, `Bugs Fixed`, `Other Changes`
https://github.com/Azure/azure-sdk/blob/26e40bb0a9444eb7c7e57d0d1be4a6cf04f3fcbb/docs/policies/releases.md#change-logs

@antkmsft can you make the changes for the new CL entries for all three packages?

File: sdk/core/azure-core/CHANGELOG.md [hunk 4]
```diff
@@ -1,11 +1,15 @@
...
+
+- Added internal support for Azure Key Vault client library.
```
Comment (ahsonkhan) [inline sdk/core/azure-core/CHANGELOG.md:11]: nit:
```suggestion
- Made internal-only changes to support the Azure Key Vault client library.
```

File: sdk/identity/azure-identity/CHANGELOG.md [hunk 5]
```diff
@@ -1,6 +1,6 @@
...
 
 ### New Features
```
Comment (ahsonkhan) [inline sdk/identity/azure-identity/CHANGELOG.md:5]:
```suggestion
### Features Added
```

File: sdk/core/azure-core/CHANGELOG.md [hunk 6]
```diff
@@ -1,11 +1,15 @@
 # Release History
 
-## 1.1.0 (Unreleased)
+## 1.1.0 (2021-07-01)
 
 ### Bug Fixes
```
Comment (ahsonkhan) [inline sdk/core/azure-core/CHANGELOG.md:5]:
```suggestion
### Bugs Fixed
```

**Review by ahsonkhan — APPROVED** [review]

**Comment by ahsonkhan** [comment]
https://github.com/Azure/azure-sdk-for-cpp/compare/af79bf2e722b3167f21fc4578457a1f5e8984234...6e4e9fd54b80e33223d50cd207879f948abd1079

**Pull request merged**